google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryRetryAlgorithm.java

```diff
@@ -55,6 +60,22 @@ public boolean shouldRetry(
       ResponseT previousResponse,
       TimedAttemptSettings nextAttemptSettings)
       throws CancellationException {
+    // Log retry info
+    int attemptCount = nextAttemptSettings == null ? 0 : nextAttemptSettings.getAttemptCount();
+    Duration retryDelay =
+        nextAttemptSettings == null ? Duration.ZERO : nextAttemptSettings.getRetryDelay();
+
+    if (LOG.isLoggable(Level.FINEST)) {
+      LOG.log(
```

Comment:
I don't see how users will see what is being retried.  Does the context for what's under retried get logged elsewhere?

Reply:
Added logging for retried method.